platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/nexus/DefaultLatestCompleteStateNexus.java

```diff
@@ -43,13 +43,13 @@ public class DefaultLatestCompleteStateNexus implements LatestCompleteStateNexus
     /**
      * Create a new nexus that holds the latest complete signed state.
      *
-     * @param stateConfig the state configuration
-     * @param metrics     the metrics object to update
+     * @param platformContext the platform context
      */
-    public DefaultLatestCompleteStateNexus(@NonNull final StateConfig stateConfig, @NonNull final Metrics metrics) {
-        this.stateConfig = Objects.requireNonNull(stateConfig);
-        Objects.requireNonNull(metrics);
+    public DefaultLatestCompleteStateNexus(@NonNull final PlatformContext platformContext) {
+        this.stateConfig = platformContext.getConfiguration().getConfigData(StateConfig.class);
+        ;
```

Comment:
;

Reply:
Fixed. Too bad spotless doesn't fix this for us.